include/MainWindow.h

```diff
@@ -420,6 +420,11 @@ class MainWindow : public QMainWindow
         void toggleRuleResultsExpanded();
         void toggleRuleResultsExpanded(bool checked);
 
+        /// Pops up a save dialog for a bash remediation
+        void genBash();
```

Comment:
write a better method name please, it's worth it for self-documentation and editors auto-complete anyway.

Reply:
also, can't we have just one method that gets "template" passed in? that way we can add even more remediation options easily later on.

Reply:
I think that it is unavoidable to have multiple methods as one menu item requires one slot method. As noted in the code, there is a potential for code reuse between those methods. I am thinking of polymorphism and I would postpone this until the remediation from results is implemented, since there could be some code reuse with this approach as well.